src/gluonts/model/wavenet/_network.py

```diff
@@ -735,7 +734,7 @@ def hybrid_forward(
         past_time_feat: Tensor,
         future_time_feat: Tensor,
         scale: Tensor,
-    ) -> Tuple[Tuple[Tensor, Tensor], Tensor]:
+    ) -> Tuple[Tuple[Tensor, Tensor], Tensor, Tensor]:
```

Comment:
For tuples of homogenous type (but unknown number of elements) we can use ellipsis, see https://docs.python.org/3/library/typing.html#typing.Tuple

```suggestion
    ) -> Tuple[Tuple[Tensor, ...], Tensor, Tensor]:
```